src/components/timeline/Event.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+interface EventProps {
+  title: string;
+  startTime: string;
+  endTime: string;
+  children: React.ReactNode;
+}
+
+const Event = ({ title, startTime, endTime, children }: EventProps): JSX.Element => {
+  const [expanded, setExpanded] = useState(false);
+
+  const toggleExpanded = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <StyledEvent>
+      <EventHeader onClick={toggleExpanded}>
+        <EventTime>{startTime}</EventTime>
+        <EventIndicator className="indicator" />
+        <EventTitle>{title}</EventTitle>
+      </EventHeader>
+
+      <EventBody hide={!expanded}>
+        <EventInnerContent>{children}</EventInnerContent>
+        <EventEndTime>Antatt sluttidspunkt: {endTime}</EventEndTime>
+      </EventBody>
+    </StyledEvent>
+  );
+};
+
+export default Event;
+
+const StyledEvent = styled.div`
+  &:hover {
+    .indicator {
+      background-color: orange;
+    }
+  }
+`;
+
+const EventTitle = styled.h3`
+  font-weight: bold;
+  border-left: 1px solid black;
+  padding: 16px 0 16px 45px;
+`;
+
+const EventHeader = styled.div`
+  display: flex;
+  position: relative;
+  h3 {
+    margin: 0px;
+  }
+`;
+
+const EventTime = styled.h3`
+  padding: 16px 30px 16px 0;
+`;
+
+const EventEndTime = styled.p`
+  font-size: 14px;
+  text-decoration: underline;
+  padding-top: 16px;
+`;
+
+const EventBody = styled.div<{ hide?: boolean }>`
+  border-left: 1px solid black;
+  padding-left: 45px;
+  margin-left: 90px;
+  max-width: 670px;
+
+  ${({ hide }) =>
+    hide &&
+    `
+    display: none;
+  `}
+`;
+
+const EventInnerContent = styled.div`
+  border-bottom: 1px solid transparent;
+
+  & > * + * {
+    margin: 16px 0;
+  }
+`;
+
+const EventIndicator = styled.div`
+  position: absolute;
+  left: 83px;
```

Comment:
I think having the indicator as a div might prove to be problematic. First of all, I'm not a huge fan of having HTML elements that are only there for styling. This is a great fit for a pseudo-element, for example.
Being pixel-specific here leads to the dot being positioned incorrectly if the size or weight of the fonts nearby are changed. A more general solution could be using `&::before` on the EventTitle, as that is where the border we want the dot to sit on is defined as well. Positioning it based on the EventTitle shouldn't be too hard either, something like `left: 0` and maybe a `transform: translateX(-50%)` could get you where you want to be.